lang/java/avro/src/main/java/org/apache/avro/data/Json.java

```diff
@@ -55,8 +56,12 @@ private Json() {}                               // singleton: no public ctor
   public static final Schema SCHEMA;
   static {
     try {
```

Comment:
We could start discussing dropping 1.6 support in the 1.9.0 release on the dev list.